### PR TITLE
Add new docs and update exisiting ones

### PR DESCRIPTION
**Pull request by NiceSapien**
You know🙏

### CODE REVIEW

**Comment by NiceSapien** [comment]
Done👍

File: app/src/main/java/com/besome/sketch/beans/ComponentBean.java [hunk 1]
```diff
@@ -153,7 +153,11 @@ public static String getComponentDocsUrlByTypeName(int type) {
 
             case COMPONENT_TYPE_FIREBASE_AUTH:
                 return "https://docs.sketchware.io/docs/component-firebase-auth.html";
-
+                case COMPONENT_TYPE_FIREBASE_CLOUD_MESSAGE:
+                 return "https://sketchware-pro/docs/components/Firebase/cloud-messaging";
```
Comment (JavkhlanK) [inline app/src/main/java/com/besome/sketch/beans/ComponentBean.java:157]: Wrong URL?

**Review by JavkhlanK — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/besome/sketch/beans/ComponentBean.java (hunk 1 above)
Comment (NiceSapien) [reply app/src/main/java/com/besome/sketch/beans/ComponentBean.java:157]: Damn

**Comment by JavkhlanK** [comment]
I think the screenshots should be improved first before merging this.

Example

**Comment by NiceSapien** [comment]
> I think the screenshots should be improved first before merging this.
> 
> Example

ok bro lemme do it😔

**Comment by NiceSapien** [comment]
> I think the screenshots should be improved first before merging this.
> 
> Example

All Done

**Review by JavkhlanK — APPROVED** [review]
Everything alright now.

**Comment by khaled-0** [comment]
wouldn't it be better to host the docs at GitHub
you know, in the hosts branch.

**Comment by ilyassesalama** [comment]
IMO, It'd be better to host the website entirely on GitHub.

**Comment by NiceSapien** [comment]
Can't have multiple domains, test deployments etc. In it, also, already configured Vercel now. Ain't switching

**Pull request merged**